modules/npm/src/main/typescript/main.d.ts

```diff
@@ -2,3 +2,5 @@ export declare function deg2hms(deg: number): string;
 export declare function deg2dms(deg: number): string;
 export declare function hms2deg(hms: string): number;
 export declare function dms2deg(dms: string): number;
+
+export declare function dateToLocalObservingNight(date: Date): string;
```

Comment:
Nice, we are sharing code!

Reply:
Yes, I think this is the most KB's anyone has ever added to check if it is before or after 14:00!